Sources/Sentry/include/HybridPublic/SentryEnvelopeItemType.h

```diff
@@ -5,3 +5,5 @@ static NSString *const SentryEnvelopeItemTypeTransaction = @"transaction";
 static NSString *const SentryEnvelopeItemTypeAttachment = @"attachment";
 static NSString *const SentryEnvelopeItemTypeClientReport = @"client_report";
 static NSString *const SentryEnvelopeItemTypeProfile = @"profile";
+static NSString *const SentryEnvelopeItemTypeReplayVideo = @"replay_video";
+static NSString *const SentryEnvelopeItemTypeReplayRecording = @"replay_recording";
```

Comment:
Can you point me in the right direction, Im not sure if I get this. Any example in the SDK of other envelope types?

Reply:
Every envelope item type also has a data category that it maps to. Just check the code of the envelope item types and the code of the data categories. Ideally, an envelope item type should also map to a data category, which we use for rate limiting and client reports. Otherwise, rate limiting and client reports won't work for your new envelope items.